### PR TITLE
[JXR-178] Update parent pom to 39

### PR DESCRIPTION
**Pull request by elharo**
Includes the usual spotless mandated reformatting

### CODE REVIEW

File: maven-jxr-plugin/src/main/java/org/apache/maven/plugin/jxr/AbstractJxrReport.java [hunk 1]
```diff
@@ -1,3 +1,21 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```
Comment (slawekjaranowski) [inline maven-jxr-plugin/src/main/java/org/apache/maven/plugin/jxr/AbstractJxrReport.java:2]: duplicate

File: maven-jxr-plugin/src/main/java/org/apache/maven/plugin/jxr/AggregatorJxrReport.java [hunk 2]
```diff
@@ -1,3 +1,21 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```
Comment (slawekjaranowski) [inline maven-jxr-plugin/src/main/java/org/apache/maven/plugin/jxr/AggregatorJxrReport.java:2]: duplicate

**Comment by slawekjaranowski** [comment]
please try:

```
git grep -c "Licensed to the Apache Software Foundation" | grep -v ":1"
```

File: maven-jxr-plugin/src/main/java/org/apache/maven/plugin/jxr/AggregatorJxrReport.java (hunk 2 above)
Comment (elharo) [reply maven-jxr-plugin/src/main/java/org/apache/maven/plugin/jxr/AggregatorJxrReport.java:2]: fixed

**Review by slawekjaranowski — APPROVED** [review]

**Pull request merged**